templates/README.md

```diff
@@ -14,8 +14,7 @@ IBMVPC_SSHKEY_ID=r134-2a82b725-e570-43d3-8b23-9539e8641944 \
 clusterctl generate cluster ibm-vpc-1 --kubernetes-version v1.14.3 \
 --target-namespace default \
 --control-plane-machine-count=1 \
---worker-machine-count=2 \
---from ~/.cluster-api/dev-repository/infrastructure-ibmvpccloud/v0.3.8/cluster-template.yaml
+--worker-machine-count=2 > capi-ibmvpc-quickstart.yaml
```

Comment:
This is not exposed for end user and lets keep this as is and will be helpful if someone wants to modify the template locally and run the some

Reply:
or may be modify this to 

```
--from ./cluster-template.yaml
```